knockout-tinymce.coffee

```diff
@@ -41,11 +42,19 @@
       # tiny mce crashes if value is null
       if value == null
         value = ""
-      if tinymce
+      if tinymce and cache != value
         if tinymce.getContent() isnt value
           tinymce.setContent value
       return
 
+  writeValueToProperty = (property, allBindings, key, value, checkIfDifferent) -> 
+    if !property or !ko.isObservable property
```

Comment:
it's honestly just a copy of the function from the original ko source that is hidden in the minified version.